projects/core/src/occ/occ-models/occ-endpoints.model.ts

```diff
@@ -0,0 +1,56 @@
+export interface OccEndpoints {
+  /**
+   * Get product details
+   *
+   * @member {string} [product]
+   */
+  product?: string;
+  /**
+   * Get reviews for a product
+   *
+   * @member {string} [productReviews]
+   */
+  productReviews?: string;
+  /**
+   * Get a product reference
+   *
+   * @member {string} [productReferences]
+   */
+  productReferences?: string;
+  /**
+   * Get a list of products and additional data
+   *
+   * @member {string} [productSearch]
+   */
+  productSearch?: string;
+  /**
+   * Get a list of available suggestions
+   *
+   * @member {string} [productSuggestings]
+   */
+  productSuggestions?: string;
+  /**
+   * Get component data
+   *
+   * @member {string} [component]
+   */
+  component?: string;
+  /**
+   * Find component data by id(s)
+   *
+   * @member {string} [components]
+   */
+  components?: string;
+  /**
+   * Get page data with list of cms content slots
```

Comment:
Do we even need to support this one? It's the same as `page`, but homepage is return if there's no page uid specified.

Reply:
All of the endpoints listed are already in use. This is just a list of what is already there.